lib/presentation/widget/shared_space_details/shared_space_details_widget.dart

```diff
@@ -219,53 +219,31 @@ class _SharedSpaceDetailsWidgetState extends State<SharedSpaceDetailsWidget> {
     );
   }
 
-  Widget _membersTabWidget(List<SharedSpaceMember> members) {
-    return ListView.builder(
-      itemCount: members.length,
-      itemBuilder: (context, index) {
-        return ListTile(
-          contentPadding: EdgeInsets.only(left: 24, top: 10, bottom: 10),
-          leading: LabelAvatarBuilder(members[index].account.name.characters.first.toUpperCase())
-            .key(Key('label_shared_space_member_avatar'))
-            .build(),
-          title: Column(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: [
-              Padding(
-                padding: EdgeInsets.only(bottom: 8),
-                child: Text(
-                members[index].account.name,
-                style: TextStyle(
-                  fontSize: 14,
-                  fontWeight: FontWeight.w500,
-                  fontStyle: FontStyle.normal,
-                  color: AppColor.loginTextFieldTextColor
-                )
-              )),
-              Text(
-                members[index].account.mail,
-                style: TextStyle(
-                  fontSize: 14,
-                  fontWeight: FontWeight.normal,
-                  fontStyle: FontStyle.italic,
-                  color: AppColor.documentModifiedDateItemTextColor
+  Widget _membersTabWidget(SharedSpaceDetailsInfo sharedSpaceDetails) {
+    return RefreshIndicator(
+        onRefresh: () async {
+          setState(() {});
```

Comment:
`setState(() {});` will make all 3 tabs be refreshed then call all corresponding APIs.